src/client/components/aea-theme/aea-menu.ls

```diff
@@ -0,0 +1,85 @@
+component-name = "aea-menu"
+Ractive.components[component-name] = Ractive.extend do
+    template: RACTIVE_PREPARSE('aea-menu.pug')
+    isolated: yes
+    data: ->
+        expand: yes
+        is-menu-open: no
+        prefer-old-menu: no
+    onrender: ->
+        __ = @
+        main-sidebar = $ @find \.main-sidebar
+        main-sidebar-button = $ @find \.main-sidebar-button
+
+        if $ window .width! > 1200
+            __.set \isMenuOpen, yes
+            main-sidebar .removeClass \collapsed
+
+        main-sidebar-button .click !->
+            if __.get \isMenuOpen
+                # close the menu
+
+                main-sidebar .addClass \collapsed
+                $ \.sub-menu-open .removeClass \sub-menu-open
```

Comment:
Frankly it cannot be applicable for per-sub-menu since we are unable to determine which sub menu is opening/closing at compile-time (even at run-time either). That's why I left them as it is.